src/contracts/core/AVSDirectory.sol

```diff
@@ -560,14 +709,13 @@ contract AVSDirectory is
             );
             // 2. rate limiting of 1 pending allocation at a time
             // read number of checkpoints after current timestamp
-            (uint224 value, uint256 pos) = _magnitudeUpdate[operator][strategy][operatorSets[i].avs][operatorSets[i]
+            (uint224 value, uint256 pos, uint256 length) = _magnitudeUpdate[operator][strategy][operatorSets[i].avs][operatorSets[i]
                 .operatorSetId].upperLookupRecentWithPos(uint32(block.timestamp));
             // no checkpoint exists if value == 0 && pos == 0, so check the negation before checking if there is a
             // a pending allocation
             if (value != 0 || pos != 0) {
                 require(
-                    pos + MAX_PENDING_UPDATES
-                        == _magnitudeUpdate[operator][strategy][operatorSets[i].avs][operatorSets[i].operatorSetId].length(),
+                    pos + MAX_PENDING_UPDATES == length,
```

Comment:
<=?

Reply:
Actually think it should be >=

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/691/commits/db66da58945da71f7f53940f6dbf656a7d25d61e

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/691/commits/decb79d6b4a33c3840a23460e55d8799b6fd455e
Fixed